src/app/common/search-input/search-input.component.html

```diff
@@ -1,5 +1,15 @@
 <div class="search-input-container d-flex">
-	<input [placeholder]="placeholder" type="text" class="form-control" [(ngModel)]="search" (keyup)="onKeyup()">
+	<input
+		type="text"
+		[placeholder]="placeholder"
+		class="form-control"
+		[(ngModel)]="search"
+		(keyup)="onKeyup()"
+		(input)="onInput()"
+	/>
 	<span *ngIf="search.length === 0" class="icon fa fa-search"></span>
 	<span *ngIf="search.length > 0" (click)="clearSearch($event)" class="icon fa fa-times"></span>
 </div>
+<div *ngIf="showMinCountMessage" [@minCount] class="min-count">
```

Comment:
Can we use the [bootstrap](https://getbootstrap.com/docs/4.5/components/forms/) recommended styling for this?

Would simplify to the following and not require any additional css.
```
<small *ngIf="showMinCountMessage" [@minCount] class="text-muted pt-2">
	Searches require a minimum of {{ minSearchCharacterCount }} characters.
</small>
```